shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -422,11 +441,25 @@ - (BOOL)flt_hasFirstResponderInViewHierarchySubtree {
       case kOpacity:
         embedded_view.alpha = (*iter)->GetAlphaFloat() * embedded_view.alpha;
         break;
-      case kBackdropFilter:
+      case kBackdropFilter: {
+        // We only support DlBlurImageFilter for BackdropFilter.
+        if (!(*iter)->GetFilter().asBlur())
+          continue;
+
+        // Sigma X is arbitrarily chosen as the radius value because Quartz only supports 1D
+        // rendering. DlBlurImageFilter's Tile Mode is not supported in CIGaussianBlurFilter so it
+        // is not used to blur the PlatformView.
+        blurRadius = @((*iter)->GetFilter().asBlur()->sigma_x());
+        [blurRadii addObject:blurRadius];
         break;
+      }
     }
     ++iter;
   }
+
+  [clipView applyBackdropFilters:blurRadii];  // TODO EMILY: pass pointer/reference?
```

Comment:
You don't have to put the TODO here, if the API of `GetFilter()` change, the PR that changes the API will have to fix it there.